streaming/src/test/java/org/apache/spark/streaming/LocalJavaStreamingContext.java

```diff
@@ -28,6 +29,7 @@
 
     @Before
     public void setUp() {
+        SparkContext$.MODULE$.stopActiveContext();
```

Comment:
Can't you just call `SparkContext.stopActiveContext()` from Java? I see static methods in the bytecode for things like `jarOfObject`.

The line is also indented incorrectly.